chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -9,6 +9,540 @@
+    { "name": "tor.bravesoftware.com", "pins": "brave"},
+    { "name": "translate.brave.com", "pins": "brave"},
+    { "name": "translate-static.brave.com", "pins": "brave"},
+    { "name": "variations.brave.com", "pins": "brave"},
+
+    // Creators
+    { "name": "creators.basicattentiontoken.org", "pins": "brave"},
+    { "name": "creators.brave.com", "pins": "brave"},
+    { "name": "publishers.basicattentiontoken.org", "pins": "brave"},
+    { "name": "publishers.brave.com", "pins": "brave"},
+
+    // Wallet
+    { "name": "goerli-infura.brave.com", "pins": "brave"},
+    { "name": "sepolia-infura.brave.com", "pins": "brave"},
+    { "name": "mainnet-infura.brave.com", "pins": "brave"},
+    { "name": "mainnet-beta-solana.brave.com", "pins": "brave"},
+    { "name": "mainnet-polygon.brave.com", "pins": "brave"},
+
+    // Rewards
+    { "name": "anonymous.ads.brave.com", "pins": "brave"},
+    { "name": "anonymous.ads.bravesoftware.com", "pins": "brave"},
+    { "name": "api.rewards.brave.com", "pins": "brave"},
+    { "name": "api.rewards.bravesoftware.com", "pins": "brave"},
+    { "name": "api.rewards.brave.software", "pins": "brave"},
+    { "name": "geo.ads.brave.com", "pins": "brave"},
+    { "name": "geo.ads.bravesoftware.com", "pins": "brave"},
+    { "name": "grant.rewards.brave.com", "pins": "brave"},
+    { "name": "grant.rewards.bravesoftware.com", "pins": "brave"},
+    { "name": "grant.rewards.brave.software", "pins": "brave"},
+    { "name": "mywallet.ads.brave.com", "pins": "brave"},
+    { "name": "mywallet.ads.bravesoftware.com", "pins": "brave"},
+    { "name": "payment.rewards.brave.com", "pins": "brave"},
+    { "name": "payment.rewards.bravesoftware.com", "pins": "brave"},
+    { "name": "payment.rewards.brave.software", "pins": "brave"},
+    { "name": "rewards.brave.com", "pins": "brave"},
+    { "name": "search.anonymous.brave.com", "pins": "brave"},
+    { "name": "search.anonymous.bravesoftware.com", "pins": "brave"},
+    { "name": "static.ads.brave.com", "pins": "brave"},
+    { "name": "static.ads.bravesoftware.com", "pins": "brave"},
+
+    // Search
+    { "name": "search.brave.com", "pins": "brave"},
+    { "name": "cdn.search.brave.com", "pins": "brave"},
+    { "name": "fg.search.brave.com", "pins": "brave"},
+    { "name": "imgs.search.brave.com", "pins": "brave"},
+    { "name": "tiles.search.brave.com", "pins": "brave"},
+    { "name": "collector.wdp.brave.com", "pins": "brave"},
+    { "name": "patterns.wdp.brave.com", "pins": "brave"},
+    { "name": "quorum.wdp.brave.com", "pins": "brave"},
+    { "name": "star.wdp.brave.com", "pins": "brave"},
+
+    // Premium
+    { "name": "account.brave.com", "pins": "brave"},
+    { "name": "account.bravesoftware.com", "pins": "brave"},
+    { "name": "account.brave.software", "pins": "brave"},
+
+    // Test page using a CA outside of the pinset (expected to be blocked)
+    { "name": "ssl-pinning.someblog.org", "pins" : "brave"}
+ ]})brave_pins_json";
```

Comment:
Missing an "end" marker:
```
      // =====END BRAVE HOSTS JSON=====
```

Reply:
Actually nevermind. It seems like iOS isn't using these markers after all. So instead of adding this one, let's just remove the other three "BEGIN" and "END" comments.

Reply:
Sure, fixed here ad9301fe80cae9dbf7f3677a0a72f536d06be074